### PR TITLE
DOC-6215: Add Vale check for any ordered lists that don't start with 1.

### PR DESCRIPTION
**Pull request by nickvigilante**
Fixes DOC-6215

### CODE REVIEW

**Comment by ianjevans** [comment]
Just did a search for what this would catch using:

~~~ shell
ag '^\s*[23456789][0-9]*\.'
~~~

And I found this false positive:

~~~
v19.1/sql-constants.md
140:3.1415
142:6.626e-34
~~~

Probably not a big deal, as it's non-blocking.

**Comment by nickvigilante** [comment]
I updated it to `^\s*[23456789][0-9]*\.\D`, so that it would skip over those. I also went ahead and just updated everything current to use 1.

**Review by mikeCRL — APPROVED** [review]
LGTM to merge after @nickvigilante explains change to the team via Slack (and perhaps after sharing vs code solution to default to 'continue' list numbering with `1.`).

File: _includes/cockroachcloud/cdc/create-core-changefeed.md [hunk 1]
```diff
@@ -11,28 +11,29 @@ In this example, you'll set up a core changefeed on your {{ site.data.products.s
 
     {% include cockroachcloud/cdc/core-csv.md %}
 
-3. Enable the `kv.rangefeed.enabled` [cluster setting](../{{site.versions["cloud"]}}/cluster-settings.html):
+
```
Comment (nickvigilante) [inline _includes/cockroachcloud/cdc/create-core-changefeed.md:14]:
```suggestion
```

**Pull request merged**